packages/itwinui-css/src/style/global.scss

```diff
@@ -35,35 +35,39 @@ html {
   }
 }
 
-.iui-body {
-  background-color: var(--iui-color-background-2);
+:where(.iui-root) {
   color: var(--iui-text-color);
   font-size: $iui-font-size;
   line-height: $iui-line-height;
 
-  @include iui-scrollbar;
   @include iui-font-family;
 
-  &.iui-no-webfonts {
-    font-family: $iui-sans-fallback;
+  // reset margin and apply background only for <body> element
+  @at-root body#{&} {
+    margin: 0;
+    background-color: var(--iui-color-background-2);
   }
-}
-
-.iui-visually-hidden {
-  @include visually-hidden;
-}
 
-[class*='iui-'],
-[class*='iui-'] * {
-  // Text highlight
-  @include iui-text-selection;
+  // border-box on all iui- elements and their descendants under iui-root, excluding any nested iui-root elements
+  :where([class*='iui-']) {
+    &:where(:not(.iui-root)),
+    * {
+      &,
+      &::before,
+      &::after {
+        box-sizing: border-box;
+      }
+    }
+  }
 
-  // Scrollbar
-  @include iui-scrollbar;
+  // text selection and scrollbar styling on iui-root and all iui- elements under it
+  &,
+  :where([class*='iui-']) {
+    @include iui-text-selection;
+    @include iui-scrollbar;
```

Comment:
Previously we were applying text-selection and scrollbar styling on _everything_ on the page (`iui-body *`). I'm not sure if we want to keep that behavior.

See the difference it already makes in our tests, a random div on the page was getting our scrollbar styles: https://itwin.github.io/iTwinUI/749/results/html_report/

Reply:
Im not sure its better to have different scrollbars in a single application :/
In that particular div situation, I think we want our scrollbar there (how it appears there, is other question).
I guess I would agree about text selection, but then again.. if users want it to be consistent, how do they do that?

Reply:
Hmm I wonder if we need to make classes for scroll area and text selection.

For now I've added `*` under `iui-root` so it won't affect anything outside at least.